CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.15.0 (2020-05-07)
```

Comment:
Should that be `3.15.1` or something else?

Reply:
Good catch. I fixed it.